<launch>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find unitree_lidar_ros)/rviz/view.rviz" />

  <node pkg="unitree_lidar_ros" type="unitree_lidar_ros_node"  name="unitree_lidar_ros_node" respawn="true">
    <rosparam command="load" file="$(find unitree_lidar_ros)/config/config.yaml"/>

  </node>

</launch>
